tests/nlu/selectors/test_selectors.py

```diff
@@ -279,17 +285,17 @@ async def _train_persist_load_with_different_settings(
 
 
 @pytest.mark.skip_on_windows
-async def test_train_persist_load(component_builder: ComponentBuilder, tmpdir: Path):
+@pytest.mark.parametrize("should_finetune", [True, False])
+async def test_train_persist_load(
```

Comment:
sorry if I've stated that wrong in the Notion document, but I think these tests might have to be one after another so that the 2nd train is actually a training which finetunes 😬